repo/migrations/Migration024.go

```diff
@@ -0,0 +1,92 @@
+package migrations
+
+import (
+	"database/sql"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path"
+)
+
+const (
+	// AM06MessagesCreateSQL - the messages create sql
+	AM06MessagesCreateSQL = "create table messages (messageID text primary key not null, orderID text, message_type integer, message blob, peerID text, url text, acknowledged bool, tries integer, created_at integer, updated_at integer);"
+	// AM06CreateIndexMessagesSQL1 - the messages index on messageID sql
+	AM06CreateIndexMessagesSQL1 = "create index index_messages1 on messages (messageID);"
```

Comment:
Please make these variable and index names meaningful. `index_messages_message_id`?

Reply:
And if we're removing the MessageByMessageID function, let's remove this index at the same time.